tests/sensors/test_sql_sensor.py

```diff
@@ -86,8 +86,8 @@ def test_sql_sensor_postgres(self):
         )
         op2.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, ignore_ti_state=True)
 
-    @mock.patch('airflow.sensors.sql.BaseHook')
-    def test_sql_sensor_postgres_poke(self, mock_hook):
+    def test_sql_sensor_postgres_poke(self):
+        mock_hook = mock.MagicMock(spec=BaseHook)
```

Comment:
I think you need to do this instead (same for all other tests)

```suggestion
    @mock.patch('airflow.sensors.sql.BaseHook', new_callable=mock.MagicMock, spec=True)
    def test_sql_sensor_postgres_poke(self, mock_hook):
```

`spec=True` is the same as `spec=BaseHook` without needing to import the class.